lib/ssh.js

```diff
@@ -191,13 +193,18 @@ SSH.prototype.start = function(options) {
             readyTimeout: self.timeout
         });
     } else if (!self.pass && self.key) {
-        self._c.connect({
-            host: self.host,
-            port: self.port || 22,
-            username: self.user,
-            privateKey: self.key,
-            readyTimeout: self.timeout
-        });
+        try {
+            self._c.connect({
+                host: self.host,
+                port: self.port || 22,
+                username: self.user,
+                privateKey: self.key,
+                passphrase: self.passphrase,
+                readyTimeout: self.timeout
+            });
+        } catch (err) {
+            return self._c.emit('error', new Error('SSH key passphrase is wrong: ' + err.toString()));
```

Comment:
Good idea but I'd make a small change here:

```javascript
self._c.emit('error', new Error('Incorrect passphrase: ' + err.toString()));
return this;
```

This ensures that the user still gets the current instance of SSH back instead of the underlying EventEmitter.